guides/basic-use/deploying.md

```diff
@@ -96,33 +96,51 @@ Lastly, if you find yourself working with an older app, you may see references t
 
 ### Content Security Policy
 
-It is a best practice to define a Content Security Policy for your application, even for local development, and to make it restrictive. That way security violations can be discovered immediately.
+It is a best practice to define a Content Security Policy (CSP) for your application and to make it restrictive. A Content Security Policy configuration defines the list of places that your app should accept data from. For example, an app may allow stylesheets from a CDN and images from a specific data storage service. By forbidding everything else common Cross-Site-Scripting (XSS) attacks could be prevented.
 
-A Content Security Policy configuration defines the list of places that your app should accept data from. For example, an app may allow stylesheets from a CDN and images from a specific data storage service.
+The [`ember-cli-content-security-policy`](https://github.com/rwjblue/ember-cli-content-security-policy) addon helps you to develop and test your application with a CSP applied. It could also inject the CSP in the app as a meta tag on build time. As any other addon it should be installed using the Ember CLI:
 
-The Content Security Policy can be set in `environment.js`:
+```sh
+ember install ember-cli-content-security-policy
+```
 
-```javascript {data-filename=config/environment.js}
+It applies a strict CSP by default, which may break your application. You could relax the policy by configuring the addon. It's configuration is managed with a `config/content-security-policy.js` file.
+
+To allow loading fronts from Google CDN the default configuration and deliver the CSP via meta tag the configuration could be changed to the following:
+
+```javascript {data-filename=config/content-security-policy.js}
 module.exports = function(environment) {
-  let ENV = {
-    ...
-    contentSecurityPolicy: {
-      'default-src': "'self'",
-      'script-src': "'self' *.my-domain-name.com",
-      'font-src': "'self' fonts.googleapis.com fonts.gstatic.com",
-      'connect-src': "'self' my-auth-provider.com",
-      'img-src': "self",
-      'style-src': "self",
-      'media-src': "self"
-    }
+  return {
+    delivery: ['meta'],
```

Comment:
Are there any other options for delivery?

Reply:
Yes. It could also be delivered using a HTTP response header. Please find details in the documentation: https://github.com/rwjblue/ember-cli-content-security-policy#configuration